v2/tools/generator/internal/astmodel/storage_package_reference.go

```diff
@@ -15,34 +15,58 @@ const (
 )
 
 type StoragePackageReference struct {
-	LocalPackageReference
+	inner   LocalPackageReference
+	version string
```

Comment:
What's the difference between this `version` and the `version` inside the `LocalPackageReference`?

ohh, this one has the storage suffix:
`version: local.version + StoragePackageSuffix`

Might be worth clarifying in a comment that `inner` is the customer facing package that this storage package is related to (or however you wanna phrase it).

Reply:
Done.